### ISSUE TITLE
airline onedark theme doesn't work

### ISSUE THREAD

**Issue by tjx666** [joshdick/onedark.vim#236]
- [x] I have carefully read and followed the [Installation section of the README](https://github.com/joshdick/onedark.vim#installation)
- [x] I have searched through the [closed issues](https://github.com/joshdick/onedark.vim/issues?q=is%3Aissue+is%3Aclosed) to see if there is an existing solution for my issue

### Terminal Emulator Info

terminal: Iterm2 3.3.12
shell: zsh

### Output From `vim --version`

```
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jun  5 2020 21:30:37)
macOS version
Included patches: 1-503, 505-680, 682-2292
Compiled by root@apple.com
```

### Issue Description

1. install onedark them using vim8 packages: 

   ```bash
   git clone git@github.com:joshdick/onedark.vim.git ~/.vim/pack/onedark/opt/onedark.vim
   ```

2. install airline using vim8 packages: 

   ```bash
   git clone https://github.com/vim-airline/vim-airline ~/.vim/pack/dist/start/vim-airline
   ```

3. put the airline onedark theme to airline themes directory using:  

   ```bash
   mv ~/.vim/pack/onedark/opt/onedark.vim/autoload/airline/themes/onedark.vim ~/.vim/pack/dist/start/vim-airline/autoload/airline/themes/
   ```

4. modify `~/.vimrc`:

   ```txt
   1 syntax on
   2 set number
   3 set smartindent
   4 set expandtab
   5 set tabstop=4
   6
   7 packadd! onedark.vim
   8 colorscheme onedark
   9 let g:airline_theme='onedark'
   ```

The airline onedark theme doesn't work:

<img width="810" alt="Snipaste_2020-09-18_02-43-02" src="https://user-images.githubusercontent.com/41773861/93513617-b0fc8980-f958-11ea-998c-e6e832b32621.png">

One thing must be pointed out is that at third step, I didn't follow README described completely.

**Comment by baktistr** [comment]
got the same isue, I used vundle instead..

the tabline is not angled like the screenshot
wonder what happens

**Comment by joshdick** [comment]
Your screenshot appears as though the airline theme is indeed working. 

The reason you don't see the angled characters that appear in the README's screenshots is because you must use a font that includes [Powerline](https://github.com/powerline/powerline) characters. Some fonts include these characters natively, but the Powerline project provides [many patched fonts](https://github.com/powerline/fonts) as well.

I'm going to close this issue since I believe I have addressed it, but feel free to keep the conversation going!

**Comment by samu-delucas** [comment]
I had the same problem and I solved it, so I'll leave my experience here for future users.

I read various threads and  issues, and this was the main answer to all of them:
>...you must use a font that includes Powerline characters.

After installing and reinstalling various fonts without success, I went back to the airline documentation, and I found the setting`airline_powerline_fonts`. From the docs:

>`* By default, airline will use unicode symbols if your encoding matches
     utf-8. If you want the powerline symbols set this variable: >
      let g:airline_powerline_fonts = 1`

Basically the solution is to include the following lines in your `.vimrc`
``` vim
let g:airline_theme='onedark'        "I assume you already had this line
let g:airline_powerline_fonts = 1    "New line!!!
```

I hope this helps :)
